src/components/FPS/index.js

```diff
@@ -170,12 +170,12 @@ const FPS = () => {
         {/* 2.3.2 chart */}
         <Text style={styles.subTitle}>2.3.2 Data Chart</Text>
         <Text style={styles.text}>A dot line chart is a common choice for FPS data because it can show changes over time or frames.</Text>
-        <View style={styles.chartDesBox} break>
+        <View style={styles.chartDesBox}>
           <View style={styles.chartContainer}><Image src={fpsImage} break /></View>
           <View style={styles.chartDesContainer}><Text style={styles.hint}>{chartDes}</Text></View>
         </View>
```

Comment:
remove 'break' maybe can cause chart and its des across two pages

Reply:
use `wrap={false}` to implement, `break` always start from new page.